tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -742,3 +742,46 @@ def test_positive_template_check_fips_enabled(
         for kind in ['PXELinux', 'PXEGrub', 'PXEGrub2', 'iPXE', 'kexec']:
             render = host.read_template(data={'template_kind': kind})['template']
             assert 'fips=1' in render
+
+    @pytest.mark.rhel_ver_match('[^6]')
+    def test_positive_verify_chronyd_timesource_kickstart_template(
+        self,
+        module_sync_kickstart_content,
+        module_target_sat,
+        module_sca_manifest_org,
+        module_location,
+    ):
+        """Read the provision template and verify that chronyd is getting installed and the host params are rendered correctly.
+
+        :id: 0c408d6f-027f-40ca-b36d-fce524877f9b
+
+        :steps:
+            1. Create a host
+            2. Read the provision template to verify host params and chronyd in rendered template.
+
+        :expectedresults: The rendered template has the host params set and chronyd is getting installed.
+
+        :Verifies: SAT-20243, SAT-19999
+
+        :parametrized: yes
+        """
+        ntp_server_value = 'server.example.com'
+        host = module_target_sat.api.Host(
+            organization=module_sca_manifest_org,
+            location=module_location,
+            name=gen_string('alpha').lower(),
+            operatingsystem=module_sync_kickstart_content.os,
+            host_parameters_attributes=[
+                {
+                    'name': 'ntp-server',
+                    'value': ntp_server_value,
+                    'parameter_type': 'string',
+                },
+            ],
+        ).create()
+        render = host.read_template(data={'template_kind': 'provision'})['template']
+        assert 'chronyd' in render
+        assert 'ntpdate' not in render
```

Comment:
Could you be specific with what you're asserting here?

Reply:
Added a comment.

Reply:
by specific, I mean if we could assert `dnf install chronyd` or assert not dnf/yum install ntpdate

Reply:
Rendered template does't have this?